src/ORM/Table.php

```diff
@@ -700,14 +700,26 @@ public function setDisplayField($field)
     public function getDisplayField()
     {
         if ($this->_displayField === null) {
+            $displayFieldFound = false;
             $schema = $this->getSchema();
             $this->_displayField = $this->getPrimaryKey();
             foreach (['title', 'name', 'label'] as $field) {
                 if ($schema->hasColumn($field)) {
                     $this->_displayField = $field;
+                    $displayFieldFound = true;
                     break;
                 }
             }
+            if (!$displayFieldFound) {
+                foreach (['title', 'name', 'label'] as $field) {
```

Comment:
I would extract the list, as you duplicate the same from above
Maybe a var above?

Other idea:
Could this be something to put into logic of bake instead?
This way the logic wouldnt have to be added here on top.

Reply:
Since the bake tool just uses the core's `->getDisplayField` if no extra option has been given to the bake command I didn't want to split up the automatic field detection between bake and the core codebase.
See https://github.com/cakephp/bake/blob/2.x/src/Command/ModelCommand.php#L626

Reply:
Really?
Doesnt bake set a field as hardcoded config into the table class?
It does for me all the time:
```php
        $this->setDisplayField('name');
        $this->setPrimaryKey('id');
```
And that would be the part where it should detect it better and set a hardcoded field to be used IMO
This way Cake core stays lean on the automatic.

Reply:
> Doesnt bake set a field as hardcoded config into the table class?

No? Why should it be hardcoded? If so, how could the tests run through with this kind of comparison file? https://github.com/cakephp/bake/blob/2.x/tests/comparisons/Model/testBakeTableConfig.php#L48

Reply:
With this PR the bake tool would be able to correctly generate 
```
$this->setDisplayField('custom_title');
```
instead of 
```
$this->setDisplayField('id');
```
which is the fallback if no displayField could be detected. Thats all I want to achieve here.